manof.py

```diff
@@ -118,7 +132,22 @@ def _register_arguments(parser):
     lift_command = subparsers.add_parser('lift', help='Provision and run targets')
     lift_command.add_argument('targets', nargs='+')
     lift_command.add_argument('--privileged', action='store_true', help='Give extended privileges to these containers')
-    lift_command.add_argument('--device', help='Add a host device to the containers')
+    lift_command.add_argument('--device',
+                              help='Add a host device to the containers (can be used multiple times)',
+                              action='append',
+                              dest='devices')
+    lift_command.add_argument('--device-cgroup-rule',
+                              help='Add a rule to the cgroup allowed devices list (e.g. c\ 42:*\ rmw)')
+    lift_command.add_argument('--device-read-bps',
+                              help='Limit read rate (bytes per second) from a device (e.g. /dev/sda:50mb)')
+    lift_command.add_argument('--device-read-iops',
+                              help='Limit read rate (IO per second) from a device (e.g. /dev/sda:50)')
+    lift_command.add_argument('--device-write-bps',
+                              help='Limit write rate (bytes per second) to a device (e.g. /dev/sda:50mb)')
+    lift_command.add_argument('--device-write-iops',
+                              help='Limit write rate (IO per second) to a device (e.g. /dev/sda:50)')
+    lift_command.add_argument('--cap-add', help='Add capability to the container', action='append')
+    lift_command.add_argument('--cap-drop', help='Drop capability from the container', action='append')
```

Comment:
Defining same arguments multiple times - Please use parent parsers to avoid duplicate definitions (Even if we didn't before)